apps/code-server/4.95.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   code:
-    image: linuxserver/code-server:4.95.3
+    image: linuxserver/code-server:4.96.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only change in this patch is that the `linuxserver/code-server:4.95.3` image has been updated to `linuxserver/code-server:4.96.1`. This version may include bug fixes or improvements, so it's good that you've upgraded. However, please note that updating to this new version should not have any significant impact on your service unless there have specific changes relevant to your configuration. If you run into issues or need more detailed guidance, feel free to ask!